docs/workflow/requirements/windows-requirements.md

```diff
@@ -51,7 +51,7 @@ These steps are required only in case the tools have not been installed as Visua
 * Install [CMake](https://cmake.org/download) for Windows.
 * Add its location (e.g. C:\Program Files (x86)\CMake\bin) to the PATH environment variable. The installation script has a check box to do this, but you can do it yourself after the fact following the instructions at [Adding to the Default PATH variable](#adding-to-the-default-path-variable).
 
-The _dotnet/runtime_ repository recommends using CMake 3.16.4 or newer, but it may work with CMake 3.15.5.
+The _dotnet/runtime_ repository requires using CMake 3.20 or newer.
```

Comment:
We should mention something in the section about the `-msbuild` flag that CMake 3.21 is required to use that flag (maybe we can add validation of that at some point?)

Reply:
That sounds good to me. Do you mean that on Windows, if you have CMake 3.21 or later, you have to always use the `-msbuild` flag? As in:

```cmd
.\build.cmd <args go here> -msbuild
```

Reply:
Not quite. I mean if you want to use the `-msbuild` flag on Windows, you need at least CMake 3.21 as the VS2022 generator doesn't exist in CMake until that version.